### ISSUE TITLE
NHS QR code not recognized in 1.1.2

### ISSUE THREAD

**Issue by marianne013** [ministero-salute/it-dgc-verificaC19-android#166]
## Describe the bug
I downloaded the VerificaC19 App today (in the UK) and I tried to scan a freshly generated NHS Covid Pass QR code (for a second vaccination, done in June). It recognizes my name and date of birth, but tells me "Certificazione non valida" without giving a reason (and screen shots are disabled due to "security policy", hmmmm). The same QR code is recognized and converted into a Digital Covid Certificate without issue by the French TousAntiCovid app.

## Expected behaviour
QR code recognized as, as far as I am aware Italy is recognizing UK NHS vaccinations now.

## Steps to reproduce the issue

Open app, click "scan QR code" button, scan NHS QR code, receive error. (It would be nice to be able to at least screen shot the error screen, even if screenshots are blocked otherwise.)

## Technical details

- Host Machine OS (Windows/Linux/Mac): Samsung Galaxy A51, Android version 11

**Comment by dgrilli** [comment]
I can confirm the bug.

Just to add the app in version 1.1.2 was working correctly last week... I think some internal update has happened that make it stop working.

Also to notice that was working only after fresh installation and not after upgrade. Another issue has been opened for this

**Comment by dgrilli** [comment]
So to add to my previous comment:

QR code expiration date: 14th of November
QR code scanned both live from the app or from download pdf.

App version 1.1.2 tested on 3 different phones (two iPhones and one android )

Is there a way to enable some debug logs to understand what is not considered valid?

**Comment by FranceBB** [comment]
I can confirm the bug.
I've been denied entry to my workplace today due to the QR Code being considered invalid.
Once the QR code was scanned, the app showed my name, surname and the date of birth, then "invalid" (the red screen) despite the code generated by the NHS App was perfectly valid, showing my second jab of the Oxford Astra Zeneca vaccine administered on the 26th of July 2021. I was also able to reproduce the bug on multiple Android mobile devices.

**Comment by mariotsi** [comment]
I can confirm this. None of the codes available on the NHS England App are now recognised. Before it was reading the code of the second dose on the app-generated PDF.

**Comment by mariotsi** [comment]
@it-eucert-team can you have a look at this? The same issue persist on the iOS version. 

It's causing quite a bit of trouble since we do not get allowed inside public places and people refuse to ready "manually" the NHS certificate even if it should be valid!

Otherwise please update the FAQ on the website so we can point the VerifcaC19 utilisers there to confirm that they should read the NHS PDF without the app if the scanning fails.

**Comment by astagi** [comment]
It's not our fault, there was a signature problem on some QR code generated by NHS. I suggest you to download a newly DCC, let me know if it works!

**Comment by mariotsi** [comment]
Thanks @astagi, I just downloaded a new set of QR codes and indeed the Dose 2 code is validated correctly as before.

Thanks for the heads up!

**Comment by astagi** [comment]
Thank you for the feedback @mariotsi, glad to hear that it works! I can close the issue, since it's not even related to the app!
